lib/data/data_sources/remote/api_service.dart

```diff
@@ -0,0 +1,46 @@
+
+import 'package:http/http.dart';
+import '../../../domain/repository/service/i_movie_service.dart';
+import 'api_const.dart';
+
+class APIMovieServiceImpl implements MovieService {
```

Comment:
Instead to this name you can use MovieServiceImpl also all the interfaces should be named like IMovieService

Reply:
addressed